scripts/rotate_vault.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
```

Comment:
Maybe you're confusing `set -x` (print commands and their arguments as they are executed) with `set -e` (exit immediately if a command exits non-zero)?  I think it's important to use `set -e` in "production" scripts like this to guard against weird edge cases in the event a command unexpectedly fails (exits non-zero).

Reply:
No, the issue is that the internal commands being executed have their own output. I'm piping that to /dev/null in some cases so it is hidden from the user and I can control the messaging. If use `set -e`, anything that fails and is piped to /dev/null is just going to be completely unknown to the user, because I have no chance to catch it, right?

Reply:
You could catch such a command with something like:

```bash
if ! mission_critical_command --opt foo >/dev/null; then
    echo "mission failed for foo"
    exit 1
fi